### ISSUE TITLE
Mongo restarting docker build error

### ISSUE THREAD

**Issue by anada10** [rnadigital/agentcloud#35]
**Describe the bug**
See https://discord.com/channels/1165866460745314304/1167801582923030538/1171861403615903784

**To Reproduce**
Docker build

**Additional context**

**Comment by krish240574** [comment]
This occurs inside my VirtualBox VM, ubuntu 22.04 - I needed to have AVX enabled in my settings, the latest vesion of mongodb kept shutting down, for want of a processor that supported AVX. 
So I chose one of two options:
1. Enable AVX inside VirtualBox(haven't gotten around to researching and doing that, crazy day)
2. Use an older version of Mongo - I used 4.2 and I'm good to go, for now. 

Will update on the AVX settings.
